examples/tests/valid/bucket_keys.w

```diff
@@ -0,0 +1,17 @@
+bring cloud;
+
+let b = new cloud.Bucket();
+
+new cloud.Function(inflight () => {
+  b.put("foo", "text");
+  b.put("foo/", "text");
```

Comment:
I did some testing and this is indeed valid on both AWS and GCP. It's surprising, but the fact that it works is a cool consequence of how object storage platforms are architected. I'm guessing it will also work on Azure.

On AWS, the UI will show the "foo/" file listed as "/" when after you open the foo directory. On Google Cloud, I wasn't able to actually see the "foo/" object listed in the UI (bug perhaps), but it was listed as a distinct file when I used the SDK to list the files in a bucket, so it seems to work the same.